src/header/common/cookie.rs

```diff
@@ -27,10 +28,15 @@ use std::str::from_utf8;
 ///    ])
 /// );
 /// ```
-#[derive(Clone, PartialEq, Debug)]
-pub struct Cookie(pub Vec<String>);
+#[allow(missing_debug_implementations)]
```

Comment:
Agreed, but with my implementation, I would have had to add a Debug trait to VecMap. I wanted to get the code out fast for sake of discussion, rather than spend too many hours on an implementation that wasn't going to be used.